type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in last_updated: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in save: No attribute 'is_valid' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save: No attribute 'cleaned_data' on tuple [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in save: No attribute 'is_valid' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_core_group_view: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_core_group_edit: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_core_settings_edit: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in sigterm: No attribute 'critical' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 32, in run: No attribute 'debug' on types.TracebackType [attribute-error]
File ""dummpy_input_file"", line 32, in run: No attribute 'debug' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in stop: No attribute 'is_alive' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _create_lb_hc: No attribute 'cs_client' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _slit_ports: Name 'mapping_parts' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _wait_if_jobid: Name 'job_result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _delete_lb_rule: No attribute '_wait_if_jobid' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in average_shifts: No attribute 'diff' on list [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in prepare_search_groups: unsupported operand type(s) for item retrieval: 'events: Generator[Any, Any, None]' and '0: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _client_config: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in match_request: Name 'match' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 47, in stream_view: Name 'emails' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 38, in mlist_view: Name 'messages' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in delete: No attribute 'model' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in XMaxSlice_BoxInput: Name 'slide' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in YMin_BoxInput: No attribute 'YRange' on float [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in YMaxSlice_BoxInput: No attribute 'YLowSlice' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in ZMinSlice_BoxInput: Name 'slide' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in ZMax_BoxInput: No attribute 'ZHigh' on float [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in ZMaxSlice_BoxInput: Name 'slide' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in SMin_BoxInput: No attribute 'correctInput' on float [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in SMax_BoxInput: No attribute 'SMax' on float [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in SMinSlice_BoxInput: No attribute 'SMax' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in CMin_BoxInput: Name 'slide' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in CMinSlice_BoxInput: Name 'slide' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in CMaxSlice_BoxInput: No attribute 'correctInput' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testgetuser: Name 'u' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testsuggestedusers: Name 'categories' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in testsuggesteduserstweets: Name 'categories' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in testme: Name 'me' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in testcreatedestroyfriendship: Name 'friend' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in __init__: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in list_hosts: Name 'host_data_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in add: Name 'tp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in draw_pie_distribution_of_elements: Name 'names_w_percent' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in draw_nullbar: Name 'rscript' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in draw_multiple_profiles: Name 'rscript' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in post: No attribute 'error' on tuple [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in get: No attribute 'redirect' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in authenticate: Name 'admin_value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_history_order: Name 'history' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 20, in _ast_scan_file_re: No attribute 'group' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __repr__: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'items' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in file_containing_import: Name 'search_paths' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in iter_dist_files: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in testupdateanddestroystatus: Name 'update' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in enc_str: No attribute 'pack' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'thriftBasicReturnQueueLock' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_cancel: Name 'channel' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in PlotFigure: No attribute 'ax' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in save_plot: No attribute 'dayofplot' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in calcdev: Name 'day' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_std_dev: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in save: No attribute 'instance' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: No attribute 'fields' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 39, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save: No attribute 'cleaned_data' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in __init__: No attribute 'fields' on List[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in display_name: Name 'name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 44, in isReady: unsupported operand type(s) for item retrieval: 'k: int' and 'slice(2, None, None): slice' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in encodeMessage: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in create_argparser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in parse_programmatic_import: unsupported operand type(s) for item retrieval: List[nothing] and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in visit_ImportFrom: No attribute 'name' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_path: No attribute 'cache_dir' on hashlib._Hash [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in _lock_file_posix: Name 'f_lock' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in unpack: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in directoryCompressedLengthTest1: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in metadataPaddingTest3: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]
File ""dummpy_input_file"", line 9, in metadataPaddingTest3: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 9, in metadataPaddingTest6: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in metadataParseTest1: Name 'compMetadata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in save: No attribute 'instance' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 29, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 42, in __init__: No attribute 'fields' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_user_login: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in XMinSlice_BoxInput: No attribute 'XHighSlice' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_MapReduce: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_Acknowledged: No attribute 'inserted_ids' on set [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_OrderedUnack_Ok: Name 'found' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in testOrderedAck_Fail: Name 'error' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in after_connection: Name 'query' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_Projection: Name 'doc' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_Sort: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_Upsert: Name 'doc' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_ReturnDocument: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_GetMoreCountWithCursor: No attribute 'assertEqual' on list [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_FindOneNone: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_ComplexCommand: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in defaultTestData: unsupported operand type(s) for item assignment: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in headerLengthTest3: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_resolve_fragment_name: Name 'key' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_loading_libraries_in_nocache: No attribute 'assertEqual' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_failure_when_setting_cache: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in setUp: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_default_cache: No attribute 'assertEqual' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_adv_cache: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_primary_key: Name 'key' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_space_compression: No attribute 'render' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_compression: No attribute 'get_name_called' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in drawVertex: Name 'point' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in nearestVertex: Name 'i' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in makePath: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Name 'layout' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in listItemClick: Name 'text' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_Upsert: Name 'docs' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_LargeData: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_CursorClosing: No attribute 'coll' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_CursorClosingWithCursor: No attribute 'extend' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_UpsertAcknowledged: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in check: Name 'parsed' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in timestamp_tz_parse: Name 'ts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in read_bytes: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in print_monthly_returns: unsupported operand type(s) for item retrieval: 'i: int' and 'i: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in LoadData: Can't assign attribute 'symbols' on str [not-writable]"
False,
True,"File ""dummpy_input_file"", line 9, in SaveSettings: Name 'SettingArray' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in LoadSettings: No attribute 'YMin_Box' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in find_assignment_in_context: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_before_christ_bc_indicator: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_timestamp_parser: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 5, in test_timestamp_with_year_over_9999: 'datetime' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 8, in message: Name 'template_name' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in subject: Name 'template_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __get_new_data: Name '_new_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in privateDataPaddingTest2: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 7, in privateDataPaddingTest3: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in unpackDirectory: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in findUniqueFileName: Name 'newFileName' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in _testHeaderLength: No attribute 'logError' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_crontab_sets_id: No attribute 'id' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_missing_jobs_removed_from_scheduler: Name 'job' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_periodic_jobs_sets_result_ttl: Name 'job_from_queue' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_periodic_job_sets_id: Name 'job_from_queue' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_periodic_job_sets_description: Name 'job_from_queue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in tearDown: No attribute 'client' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in distance_from_zero: Name 'no' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in rental_car_cost: Name 'tuto' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in trip_cost: Name 'notreal' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in average: Name 'total' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in count: No attribute '__iter__' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in purify: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_image: Name '_cli' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_notification_id_1: No attribute 'client' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_auth_set_user_auth: Name '_cli' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in assertParser: No attribute 'append' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_imgur_credentials_env: No attribute 'make_env' on Tuple[str, str, str, str, None] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_album_images: No attribute 'assertParser' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in validate_email_with_name: Name 'recipient' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in validate_template_syntax: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'supporterBrokers' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in __init__: No attribute 'broker' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in consume: Name 'reply' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_cache_backend: No attribute 'get_template_key' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_version: No attribute 'group' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_delete_coll: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in test_create: No attribute 'uuid_field' on Dict[str, Optional[Union[bool, float, int, str]]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_form: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_missing_fields: Name 'field' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_form: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in set_context_data: No attribute 'user' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in set_context_data: Can't assign attribute 'descriptions' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get: No attribute 'template_name' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'bot' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_album_create: No attribute 'assertParser' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_album_set_images: No attribute 'assertParser' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_comment_reply: No attribute 'client' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_conversation_create: No attribute 'cli' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_gallery_item_tags: No attribute 'client' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_gallery_comments: No attribute 'cli' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_enqueue_job: Name 'scheduler' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in delete_timeentries_marked_for_delete: Name 'TimeEntry' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in save: No attribute 'Role' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in save: No attribute 'task' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 17, in _make_api_request: No attribute 'status_code' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in generateColorMap: Function struct.pack was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 13, in levenshtein: unsupported operand type(s) for item assignment: range [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in isfloat: Name 'a' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in isint: Name 'a' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in stSpectralCentroidAndSpread: Name 'ind' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in stSpectralRollOff: Name 'totalEnergy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 49, in ref_table: Name 'model_name' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 43, in ref_table: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in render: No attribute 'replace' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in median: Name 'b' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in assert_not_raise: Name 'msg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_if_oppucation_populated_from_choices: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_filling_file_field: No attribute 'dummy' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_uses_generator_defined_as_string_on_settings_for_custom_field: No attribute 'assertEqual' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in test_channel_lookup: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_scheduled_time: No attribute 'assertEqual' on datetime.datetime [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in lookup: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _load_page: Name 'page' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in stMFCC: Name 'mspec' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in oauth_authorized: Name 'next_url' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in request: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in save_weights: Name 'idx' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in save_momentums: unsupported operand type(s) for item retrieval: 'ind: int' and 'ind: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_list_products: No attribute 'assertEqual' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create_product_transaction: Name 'trx_obj' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in cancel_product_transaction: Name 'trx_obj' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in cancel_purchases: No attribute 'count' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _testTableDirectoryTableOrder: Name 'storedOrder' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _shouldSkipMetadataTest: Name 'metaOffset' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in _testMetadataPadding: Name 'padding' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _testMetadataParse: Name 'metadata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in private_alarms: Name 'server' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in _get_modules: No attribute 'path' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 33, in core_logo_content: No attribute 'read' on str [attribute-error]
File ""dummpy_input_file"", line 33, in core_logo_content: unsupported operand type(s) for +: str and bytes [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in number_format: No attribute 'index' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in permission_block: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in tags_box: Name 'new_tags' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in original: Name 'real_mod' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __str__: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in get: No attribute 'cancel' on Callable[[], Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in wait: Name 'timer' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _process_content_header: Name 'method' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in run: Name 'seg_service' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in inner: No attribute '_assert_valid_value_and_cast' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_user_without_role: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_not_allowed_permission: Name 'user' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_revoke_revoked_permission: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_has_role_tag: No attribute 'tag_test' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in test_numeric_type_validates_and_casts_decimal: No attribute 'value' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_other_value_not_numeric: No attribute 'assertRaisesRegexp' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_instantiate: No attribute 'assertRaisesRegexp' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in done: Name 'context' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in pytest_addoption: Name 'group' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in classify: Name 'pos_prob' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_relevant_features: Name 'neg_dump' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 17, in test_headers: No attribute 'get' on asyncio.streams.StreamReader [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in handles: Name 'handles' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_file_head: No attribute 'read' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in libusb_set_iso_packet_lengths: Name 'iso_packet_desc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in check_master: Name 'cur_master' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 22, in main: No attribute 'add_option' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute 'product_id' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in move: Name 'rel_pos' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_slack_user_id: Name 'slack_user_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in containers: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in loopbox: Name 'l' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in alarm_key_auth: Name 'container' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 50, in domains: Name 'j' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 35, in tags: Name 'j' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 48, in domain: Name 'new_tags' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in private_loopboxes: Name 'j' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in private_domains_rsa: No attribute 'domain_set' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 26, in private_metrics_domain_do: Name 'm' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_random_from_list: Name 'random_number' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_synonym: Name 'synonyms_for_word' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Name 'subdomain_double_hyphens_converted' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in random_css: Name 'css' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in parse: Name 'token' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in open_log_file: No attribute 'vim' on str [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_nse_opener: Name 'opener' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in resolveRootNode: No attribute 'resolveNodeRelations' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in dict_from_json_file: Name 'json_file' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_multiline_headers: No attribute 'loop' on bytes [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 14, in test_invalid_headers: No attribute 'loop' on asyncio.tasks.Task [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_no_body: No attribute 'body' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test: Name 'matcher' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_bad_version: Name 'task' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_args: Name 'args' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in get_args: Name 'args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in main: Name 'ds_obj_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _send_request: Name 'res' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_alarm_refs: Name 'alarm_state' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in connect: 'getpass' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 4, in get_first_level_of_vm_folders: Name 'children' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in list_graphs: No attribute 'plugins' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in wrapper: No attribute 'is_authenticated' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in expanded: Name 'in_init' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in get_mark: Name 'char' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in update_mark: Name 'widget' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in change_focus: No attribute 'get_focus' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 38, in load: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in load_citekeys: Function json.load was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in parse_val: No attribute 'groups' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in make_target: Name 'target_uri' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testChainAfterAddLink: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testLoadsSingleEmbeddedObject: Name 'foo' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testLoadsLinksInEmbeddedObject: Name 'link' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in embed: Name 'new_embed' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in testStartsWithNoProperties: Name 'doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in profile_periodically: Name '__' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _trace: Name 'target' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __exit__: No attribute 'stop' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in markup_stats: No attribute 'name' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in remove_remote: Name 'cmd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in get_current_rev: No attribute 'close' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in validate_id: Name 'normalized_vat_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in main: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 30, in main: Name 'vmfolder' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_args: Name 'my_args' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in main: Name 'si' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in setup_args: Name 'my_args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in build_schema_query: Name 'schema_query' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 46, in wrapped_function: unsupported operand type(s) for item assignment: None [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in testDeleteEmbedWithNoEmbeds: Name 'target_doc' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in testSetCurie: Name 'new_doc' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testDropCurie: Name 'new_doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Can't assign attribute 'description' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'citation' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 19, in download: Name 'url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 21, in setup_module: Name 'script_file' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in get_script_module: Function imp.load_module was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in encrypt: Function _Hash.update was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: No attribute '_max' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testEmbeddedObjectHasParentCuries: Name 'sizing_doc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in testAddSimpleLink: No attribute 'links' on Dict[str, Dict[str, Dict[str, str]]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in testAddWrappedLink: No attribute 'add_link' on Dict[str, Dict[str, Union[Dict[str, str], List[Dict[str, str]]]]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in testDeleteOnlyLinkForRel: No attribute 'assertFalse' on Dict[str, Dict[str, Dict[str, str]]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in testDeleteIndividualLinks: No attribute 'delete_link' on Dict[str, Dict[str, Union[Dict[str, str], List[Dict[str, str]]]]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in testEmbed: No attribute 'embedded1' on Dict[str, Dict[str, Any]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in testEmbedAnotherRel: No attribute 'embedded2' on Dict[str, Dict[str, Any]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in testEmbedSameRel: No attribute 'assertEquals' on Dict[str, Dict[str, list]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in make_doc: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _cg_str: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in query: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in catalogs: unsupported operand type(s) for item assignment: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_known_id: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in item_keys: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in keyfunc: Name 'keyvalue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in main: Function TextTestRunner.run was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in branch: No attribute 'readline' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in perfect_hash: Name 'offset' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in encode: Name 'size' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in loop: No attribute 'unhandled_input' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in update_result: Name 'cpu_time' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in ACEs: Name 'a' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in sds_entries: No attribute 'offset' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_size: Name 'data_attribute' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in MODULE_LIST: Function imp.load_module was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in insert_statement: Name 'statement' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in name_matches: Name 'max_ratio' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_help: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_basic_linting: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in add_source_files_to_schema: Name 'db' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_sample_data_over_time_2: No attribute 'percentile' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 23, in test_percentile_algorithm_extremes: No attribute 'percentile' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_meta: No attribute 'assertTrue' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get: Name 'f' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in set: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in then: Name 'analysis' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in index: Name 'up_token' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 17, in _dump_item: Name 'l' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in dump: Name 'i' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _main: No attribute 'vsParse' on BinaryIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in unpack_guid: Name '_bin' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in new: Name 't' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'kwargs' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _http_check_json: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in from_string: Name 'port' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in mimetype: Name 'mimetype' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in save: Can't assign attribute 'size' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in right_dict: Name 'cache' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in save: No attribute 'reltype' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_queryset: Name 'qs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get: No attribute 'content_type' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in list: No attribute 'connection' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in camel_case: Name 'first_pass' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in make_timestamp: Name 'struct' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_datetime: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in ascii_strings: No attribute 'group' on re.Pattern [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in make_standard_information_model: Name 'ret' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in make_attribute_model: No attribute 'allocated_size' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in unpack_string: Name 'o' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in slack: Name 'end' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in end_offset: Name 'string_end' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in run: Name 'offset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in find_connection: No attribute 'find_connections' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in repository: No attribute 'addfinalizer' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in progress_update: Name 'skip' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __del__: No attribute '_volume_offset_text' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 29, in send_ready: Name 'error' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in query_nsqd: Name 'address' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in query_lookupd: Name 'msg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 20, in start_backoff: Name 'error' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_auth_config_json: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in on_input_name: No attribute 'settings' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in shrink_if_number: Name 'cvt' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: No attribute 'group' on None [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in process_contours: unsupported operand type(s) for item assignment: int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in arr_from_img_storage: Name 'arr' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in onrightof: Name 'right' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in above: Name 'left' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in when: No attribute '__selectors' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in create_gist: Name 'file_full_name' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in rename_gist: Name 'old_filename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 33, in choose_gist: No attribute 'extend' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in on_input_name: Name 'message' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in on_input_name: Name 'message' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_hello_version: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init_metastore: No attribute '__newctime' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in truncate: Can't assign attribute 'dirty' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 4, in write: No attribute 'buf' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in h2ip: Name 'ips' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in check_wpsymposium: Name 'site' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in xilinx7_reader: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __insert: No attribute '__getctx' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __verify_write: Name 'saved_value' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __cache_check_gc: No attribute 'cached_nodes' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __report_top_blocks: Name 'row' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
